include/fmt/compile.h

```diff
@@ -538,14 +552,8 @@ template <typename Char, typename T, int N> struct field {
 
   template <typename OutputIt, typename... Args>
   constexpr OutputIt format(OutputIt out, const Args&... args) const {
-    if constexpr (is_named_arg<typename std::remove_cv<T>::type>::value) {
-      const auto& arg = get<N>(args...).value;
-      return write<Char>(out, arg);
-    } else {
-      // This ensures that the argument type is convertile to `const T&`.
-      const T& arg = get<N>(args...);
-      return write<Char>(out, arg);
-    }
+    const T& arg = get_arg_checked<T, N>(args...);
+    return write<Char>(out, arg);
```

Comment:
Since we check convertibility in `get_arg_checked` the `arg` variable is not longer needed. Please remove it combining the two statements into one.

Reply:
Done